flamingpy/utils/viz.py

```diff
@@ -619,7 +773,7 @@ def syndrome_plot(code, ec, index_dict=None, drawing_opts=None):
             "show_axes",
         ]
         egraph_opts = {k: drawing_opts[k] for k in egraph_args}
-        fig, ax = draw_EGraph(code.graph, **egraph_opts)
+        fig, ax = draw_EGraph_matplotlib(code.graph, **egraph_opts)
```

Comment:
I'm not sure if this is the best solution. Should we offer a plotly backend here too?